bridge/talk/talk.go

```diff
@@ -0,0 +1,114 @@
+package talk
```

Comment:
As talk is too generic, I think we should better rename this bridge and package to `nctalk`